ETANotifications/Services/NotificationService.cs

```diff
@@ -21,14 +25,14 @@ public NotificationService()
             _twilioRestClient = new TwilioRestClient(_accountSid, _authToken);
         }
 
-        public NotificationService(TwilioRestClient twilioRestClient)
-        {
-            _twilioRestClient = twilioRestClient;
-        }
-
         public void SendSmsNotification(string phoneNumber, string message, string statusCallback)
         {
-            _twilioRestClient.SendMessage(_twilioNumber, phoneNumber, message, statusCallback);
+            TwilioClient.SetRestClient(_twilioRestClient);
```

Comment:
if you `TwilioClient.Init(...)` in the constructor of this class then this line is no longer necessary here